### PR TITLE
More template changes

### PR DESCRIPTION
**Pull request by bbelderbos**

### CODE REVIEW

File: serialize_json/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[package]
```
Comment (bbelderbos) [inline serialize_json/Cargo.toml:1]: needed (as well as `src/` folder) to run the tests locally :)

File: serialize_json/exercise.md [hunk 2]
```diff
@@ -1,12 +1,7 @@
 ---
 Level (easy, medium, hard): medium
 Tags (2-5): json, strings
-GitHub to credit (leave blank if existing PR user): https://github.com/Tanner-Ray-Martin
-
-[dependencies]
-serde = { version = "1.0", features = ["derive"] }
-serde_json = "1.0"
-
+GitHub to credit: https://github.com/Tanner-Ray-Martin
```
Comment (bbelderbos) [inline serialize_json/exercise.md:4]: made required field

**Review by Tanner-Ray-Martin — APPROVED** [review]
I got it now :)

**Comment by bbelderbos** [comment]
Thanks

**Pull request merged**